tests/README.md

```diff
@@ -20,9 +20,39 @@ This script creates `versionsTest` directory to which zstd repository is cloned.
 Then all tagged (released) versions of zstd are compiled.
 In the following step interoperability between zstd versions is checked.
 
+#### `automated-benchmarking.py` - script for benchmarking zstd prs to dev
+
+This script benchmarks facebook:dev and changes from pull requests made to zstd and compares
+them against facebook:dev to detect regressions. This script currently runs on a dedicated
+desktop machine for every pull request that is made to the zstd repo but can also
+be run on any machine via the command line interface.
+
+There are three modes of usage for this script: mode 0 will fetch all open pull requests
+in the repo and benchmark them against dev, mode 1 will fetch the pull request
+associated with the current hash and benchmark that against dev, and mode 3 will
+continuously run mode 1 when a new pull request is added or when an existing pull
+request gets a new commit.
+
+```
+Example usage: python automated_benchmarking.py fuzz "1,2,3" "0" "1" ""
+```
+
+```
+usage: automated_benchmarking.py [-h] directory levels mode emails
+
+positional arguments:
+  directory   directory with files to benchmark
+  levels      levels to test eg ('1,2,3')
+  mode        0: regular, 1: pull request ci, 2: continuous
```

Comment:
`mode` : Is there a way to use _names_ instead of numbers here ?
That would make the associated command line much more readable, hence more maintainable.

Reply:
Took me a second to remember what these modes were after not looking at this code for a week so yes, good call:) I'll give them labels.

Reply:
Should the doc be updated ? It seems numbers are no longer useful for `mode`.